summary.md

```diff
@@ -0,0 +1,10 @@
+---
+layout: page
+title: Consent Form Builder Discovery Summary
+---
+## Background 
+The team identified problems with Barnardo's processes for developing consent forms. These included the amount of time spent filling in forms, unfriendly language and the lack of an easy storage and management system.
+
+The team were further aiming to get a useful and usable product out the door, quickly, to make friends and influence people.
+
+A wider iteration of the tool is being scoped that takes into account wider organisational needs and the effect of GDPR. 
```

Comment:
would it be possible to link to GDPR or at least provide an explanation to the acronym?